src/framework/midi/midievent.h

```diff
@@ -136,7 +136,8 @@ struct Event {
     static Event fromRawData(const uint32_t* data, size_t count)
     {
         Event e;
-        memcpy(e.m_data.data(), data, std::min(count, e.m_data.size()));
+        size_t numBytes = std::min(count, e.m_data.size()) * sizeof(uint32_t);
```

Comment:
Better `sizeof(data)`?

Reply:
No, `sizeof(data)` would be the size of a pointer (to uint32_t), which equals 8, on 64bit systems. 
`sizeof(*data)` or `sizeof(data[0])`  would have been an option, but I don't think it would be _much_ better.

Reply:
That's what I meant, it it would be better, as then the data type could change without the code breaking